main.go

```diff
@@ -1,7 +1,26 @@
 package main
 
-import "github.com/open-feature/flagd/cmd"
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/open-feature/flagd/cmd"
+)
+
+var (
+	version = "dev"
+	commit  = "HEAD"
+	date    = "unknown"
+)
 
 func main() {
-	cmd.Execute()
+	showVersion := flag.Bool("version", false, "show version")
```

Comment:
this could be added in as a cobra command to keep things consistent

```
// versionCmd represents the version command
var versionCmd = &cobra.Command{
	Use:   "version",
	Short: "Get Flagd version",
	Long:  ``,
	Run: func(cmd *cobra.Command, args []string) {
		fmt.Printf("flagd %s (%s) built at %s\n", version, commit, date)
		os.Exit(0)
	},
}

// added into either `Execute()` or `init()`
rootCmd.AddCommand(versionCmd)
```

then can be run with `flagd version`

Reply:
Will see what he thinks in https://github.com/open-feature/flagd/issues/138